ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -0,0 +1,229 @@
+  const buttonTitle = () => {
+    if (isEmailValidated) {
+      return I18n.t("global.buttons.continue");
+    } else {
+      if (isValidateEmailButtonDisabled) {
+        return I18n.t("email.newvalidate.buttonlabelsent");
+      } else {
+        return I18n.t("email.newvalidate.buttonlabelsentagain");
+      }
+    }
+  };
+
+  // this function contol if the button is disabled. It is disabled if the email is sent and the timeout is active
+  const isButtonDisabled = () => {
+    if (isEmailValidated) {
+      return false;
+    } else {
+      return isValidateEmailButtonDisabled;
+    }
+  };
+
+  const handleSendEmailValidationButton = () => {
+    if (isEmailValidated) {
+      hideModal();
+    } else {
+      // send email validation only if it exists
+      pipe(
+        optionEmail,
+        O.map(_ => {
+          sendEmailValidation();
+        })
+      );
+    }
+  };
+
+  const navigateToInsertEmail = () => {
+    if (isOnboarding) {
+      hideModal();
+      NavigationService.navigate(ROUTES.ONBOARDING, {
+        screen: ROUTES.ONBOARDING_INSERT_EMAIL_SCREEN
+      });
+    } else {
+      hideModal();
+      NavigationService.navigate(ROUTES.PROFILE_NAVIGATOR, {
+        screen: ROUTES.INSERT_EMAIL_SCREEN
+      });
+    }
+  };
+
+  const renderFooter = () => (
+    <FooterWithButtons
+      type={"SingleButton"}
+      leftButton={{
+        testID: "button-test",
+        block: true,
+        bordered: !isEmailValidated,
+        disabled: isButtonDisabled(),
+        onPress: handleSendEmailValidationButton,
+        title: buttonTitle()
+      }}
+    />
+  );
+
+  useEffect(() => {
+    // use polling to get the profile info, to check if the email is valid or not
+    // eslint-disable-next-line functional/immutable-data
+    polling.current = setInterval(() => reloadProfile(), profilePolling);
+    return () => {
+      hideModal();
+      clearTimeout(timeout.current);
+      clearInterval(polling.current);
+    };
+  }, [hideModal, reloadProfile]);
+
+  useEffect(() => {
+    // send validation email KO
+    if (pot.isError(emailValidation.sendEmailValidationRequest)) {
+      IOToast.error(I18n.t("global.actions.retry"));
+      // send validation email OK
+    } else if (pot.isSome(emailValidation.sendEmailValidationRequest)) {
+      IOToast.show(I18n.t("email.newvalidate.toast"));
+      setIsValidateEmailButtonDisabled(true);
+      // eslint-disable-next-line functional/immutable-data
+      timeout.current = setTimeout(() => {
+        setIsValidateEmailButtonDisabled(false);
+      }, emailSentTimeout);
+    }
+  }, [emailValidation.sendEmailValidationRequest]);
+
+  useEffect(() => {
+    if (isEmailValidated) {
+      clearInterval(polling.current);
+    }
+  }, [isEmailValidated]);
+
+  return (
+    <TopScreenComponent
+      goBack={false}
+      accessibilityEvents={{ avoidNavigationEventsUsage: true }}
+    >
```

Comment:
```suggestion
    <TopScreenComponent
      goBack={false}
      contextualHelpMarkdown={{
        title: "email.validate.title",
        body: "email.validate.help"
      }}
      accessibilityEvents={{ avoidNavigationEventsUsage: true }}
    >
```

To show the contextual help icon, once the e-mail's been verified.

Reply:
I had to change components to insert the props you mentioned because the other one was giving display problems. Anyway, I'm done!